spec/unit/packagers/msi_spec.rb

```diff
@@ -316,6 +336,23 @@ module Omnibus
       end
     end
 
+    describe '#fast_msi' do
+      it 'is a DSL method' do
+        expect(subject).to have_exposed_method(:fast_msi)
+      end
+
+      it 'requires the value to be a TrueClass or a FalseClass' do
+        expect {
+          subject.fast_msi(Object.new)
+        }.to raise_error(InvalidValue)
+      end
+
+      it 'returns the given value' do
+        subject.fast_msi(true)
+        expect(subject.fast_msi).to be_truthy
+      end
+    end
+
```

Comment:
We need to add some unit test coverage for `Omnibus::Packager::MSI#zip_command` and `Omnibus::Packager::MSI#heat_command`. We need to verify the correct version of the command is returned when `Omnibus::Packager::MSI#fast_msi` is true AND that the command parameters are correct.